src/common.jl

```diff
@@ -114,10 +115,10 @@ function Base.setproperty!(cache::SampledIntegralCache, name::Symbol, x)
 end
 
 function SciMLBase.init(prob::SampledIntegralProblem,
-    alg::SciMLBase.AbstractIntegralAlgorithm;
+    alg::SciMLBase.AbstractIntegralAlgorithm; cumulative = false,
```

Comment:
that breaks type-stability, it needs to be type level information.

Reply:
so, we should do `cumulative::Bool = false` to make it type stable?

Reply:
No, it would need to be a value type, `Val(false)`

Reply:
Ok, addressed in [46a11e1](https://github.com/SciML/Integrals.jl/pull/182/commits/46a11e10c8a29578d00d27df01202700ffb1d117)